docs/setup/prerequisites/runtime.md

```diff
@@ -12,6 +12,7 @@ This page provides an outline of how to use several common container runtimes wi
 - [docker](#docker-engine)
 - [Kata containers](#kata-containers)
 - [Virtlet](#Virtlet)
+- [iSulad](#iSulad)
```

Comment:
```suggestion
- [iSulad](#isulad)
```
It needs to be in lowercase, otherwise it won't jump

Reply:
ok